nilearn/plotting/surface/_matplotlib_backend.py

```diff
@@ -120,6 +121,41 @@ def _adjust_plot_roi_params(params):
         params["cbar_tick_format"] = "%i"
 
 
+def _normalize_bg_data(data):
+    """Normalize specified ``data`` and return.
+
+    Parameters
+    ----------
+    data : :obj:`numpy.ndarray`
+        An array containing surface data
+
+    Returns
+    -------
+    data : :obj:`numpy.ndarray`
+        An array containing surface data
+    """
+    vmin, vmax = np.min(data), np.max(data)
```

Comment:
Here instead of `np.min`, shall we use `np.nanmin`? The same is valid for max.

Reply:
I thinks it would be better.